packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -31,7 +32,8 @@ class CryptoWatchPriceFeed extends PriceFeedInterface {
     getTime,
     minTimeBetweenUpdates,
     invertPrice,
-    decimals = 18
+    decimals = 18,
+    ohlcPeriod = 60 // One minute is CryptoWatch's most granular option.
```

Comment:
Changed the price feed to make this configurable. by default still uses 1 min resolution but can be tuned to accommodate any different duration. for example, developer mining does not need more precision than once per hour(even then that's very granular).